<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>753</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="circuitGraphicsView">
      <property name="styleSheet">
       <string notr="true">background-color: white;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulate">
    <property name="title">
     <string>Simulate</string>
    </property>
    <addaction name="actionTransient"/>
    <addaction name="actionAcSweep"/>
    <addaction name="actionPhaseSweep"/>
   </widget>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuNode_library">
    <property name="title">
     <string>&amp;Node library</string>
    </property>
    <addaction name="actionOpenNodeLibrary"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuSimulate"/>
   <addaction name="menuNode_library"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAddWire"/>
   <addaction name="actionAddResistor"/>
   <addaction name="actionAddCapacitor"/>
   <addaction name="actionAddInductor"/>
   <addaction name="actionAddGND"/>
  </widget>
  <action name="actionTransient">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transiet</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionAcSweep">
   <property name="text">
    <string>AC Sweep</string>
   </property>
  </action>
  <action name="actionPhaseSweep">
   <property name="text">
    <string>Phase Sweep</string>
   </property>
  </action>
  <action name="actionAddResistor">
   <property name="text">
    <string>Resistor</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddCapacitor">
   <property name="text">
    <string>Capacitor</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddInductor">
   <property name="text">
    <string>Inductor</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="shortcut">
    <string>L</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddVdc">
   <property name="text">
    <string>DC Voltage Source</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddVsin">
   <property name="text">
    <string>Sinusoidal Voltage Source</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddIdc">
   <property name="text">
    <string>DC Current Source</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddIsin">
   <property name="text">
    <string>Sinusoidal Current Source</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionOpenNodeLibrary">
   <property name="text">
    <string>Component Library...</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddGND">
   <property name="text">
    <string>GND</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAddWire">
   <property name="text">
    <string>Wire</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
